docs/addons.md

```diff
@@ -139,6 +141,7 @@ spec:
       enabled: true
       memoryRequest: 5Mi
       cpuRequest: 25m
+      image: "k8s.gcr.io/dns/k8s-dns-node-cache:1.20.0"
```

Comment:
Even if we support setting image, we shouldn't encourage people to do so in the docs.

Reply:
true that ^ changing